dash_table/__init__.py

```diff
@@ -13,7 +13,7 @@
     _sys.exit(1)
 
 _basepath = _os.path.dirname(__file__)
-_filepath = _os.path.abspath(_os.path.join(_basepath, 'package.json'))
+_filepath = _os.path.abspath(_os.path.join(_basepath, 'package-info.json'))
```

Comment:
Whenever we want to include new non-Python files in the distribution, we need to add them to `MANIFEST.in`:
https://github.com/plotly/dash-table/blob/10b80296a8577a3efae20f996e71c2917a4d7e51/MANIFEST.in#L1-L5

Python packaging is a white-list rather than a blacklist like `.npmignore`.

Reply:
Ok cool, missed this comment when I asked at top level.